package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sdformat_urdf</name>
```

Comment:
(Feel free to dismiss) As the peanut gallery, can't say I love this name... Some other candidates:

- `sdformat_urdf_converter`
- `sdformat_to_urdfdom`
- `sdformat_urdfdom`
- `sdformat_urdfdom_plugin`

Reply:
To add my own thoughts to this bikeshed:

I don't think we should use the name "urdfdom" in it, as to me, that implies converting to a URDF XML format (which is not what we are up to here).  I'm fine with:

* `sdformat_urdf`
* `sdformat_urdf_plugin`
* `sdformat_to_urdf`

Reply:
I like `sdformat_to_urdf`

Reply:
It doesn't seem like any of the names are obviously winners. At least `sdformat_urdf` matches `collada_urdf`, so I'm inclined to leave it as is for consistency.